web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -184,7 +184,15 @@ export namespace TaskItem {
       if (this.customAriaLabel) {
         return this.customAriaLabel;
       }
-      return `${this.mediaType} ${this.status} ${this.title} ${this.queue} ${this.quantity ? this.quantity : ""} ${
+      let queueContent = this.queue;
+
+      if (!queueContent) {
```

Comment:
Please write unit test for this to cover this lines

Reply:
Done